### PR TITLE
DOCS-208 - Connect Elastic Quickstart not working

### PR DESCRIPTION
**Pull request by joel-hamill**
https://confluentinc.atlassian.net/browse/DOCS-208

- related https://github.com/confluentinc/kafka-connect-elasticsearch/pull/169

Staging server http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/693/3.3.0/connect/connect-elasticsearch/docs/elasticsearch_connector.html#quick-start

### CODE REVIEW

**Comment by joel-hamill** [comment]
@ewencp @kaiwaehner i haven't been able to actually get this to work locally. can you take a look? the staging server is [here](http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/693/3.3.0/connect/connect-elasticsearch/docs/elasticsearch_connector.html#quick-start).

**Comment by kaiwaehner** [comment]
Hi Joel,

I checked your link...

I can only confirm that it worked for me using standalone mode started via regular Kafka script. I did not start it using Confluent CLI and distributed mode. I think this did not work for me either (IIRC).

Also not sure, but probably best to start Elastic as first step (not sure if you did)...

Kai

**Comment by joel-hamill** [comment]
@kaiwaehner 
>> I did not start it using Confluent CLI and distributed mode. 
in your comments [here](https://confluentinc.atlassian.net/browse/DOCS-208?focusedCommentId=43764&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-43764) didn't you use the CLI to start Schema Registry; and start Kafka Connect as standalone? my instructions are:
1. Start Schema Registry, Kafka, ZK using the CLI
2. Start Kafka Connect as standalone
...

**Comment by ewencp** [comment]
@joel-hamill I tagged the connect team for review, they should be able to help review and get this into a good state.

**Comment by kkonstantine** [comment]
Why are we advising in the quickstart to use a mix of CLI and connect-standalone?

My impression is that the old quickstart just contained a stray reference to `connect-standalone` that we should remove. But what is wrong with using the CLI from start to finish like we do with our other connectors?

Asking just before trying to run the quickstart with the CLI. I believe that if there's something buggy there we just need to fix that, instead of using standalone connect.

**Comment by joel-hamill** [comment]
@kaiwaehner @kkonstantine is there a known Elastic issue for 6.x that i can link to for more information?

File: docs/elasticsearch_connector.rst [hunk 1]
```diff
@@ -26,44 +26,71 @@ connector provides a feature to infer mapping from the schemas of Kafka messages
...
+To use Elasticsearch you will need Schema Registry, |zk|, Kafka, and Kafka Connect.
 
-   $ confluent start
+#.  Start Schema Registry using the Confluent CLI. This also starts its dependencies, |zk| and Kafka.
 
-Every service will start in order, printing a message with its status:
+    .. sourcecode:: bash
 
-.. sourcecode:: bash
+        $ confluent start schema-registry
```
[the remark this answers is not in the document]
Comment (joel-hamill) [reply docs/elasticsearch_connector.rst]: CP installed and running is now listed as a prerequisite (like the other quick starts).

File: docs/elasticsearch_connector.rst [hunk 2]
```diff
@@ -26,44 +26,71 @@ connector provides a feature to infer mapping from the schemas of Kafka messages
...
+
+    .. sourcecode:: bash
+
+        Starting zookeeper
+        zookeeper is [UP]
+        Starting kafka
+        kafka is [UP]
+        Starting schema-registry
+        schema-registry is [UP]
+
```
[the remark this answers is not in the document]
Comment (joel-hamill) [reply docs/elasticsearch_connector.rst]: CP installed and running is now listed as a prerequisite (like the other quick starts).

File: docs/elasticsearch_connector.rst [hunk 3]
```diff
@@ -73,101 +100,100 @@ Then in the console producer, type in:
...
+            $ ./bin/connect-standalone etc/schema-registry/connect-avro-standalone.properties \
+            etc/kafka-connect-elasticsearch/quickstart-elasticsearch.properties
+
+
+#.  After the connector finishes ingesting data to Elasticsearch, check that the data is available in Elasticsearch:
+
+    .. sourcecode:: bash
+
+      $ curl -XGET 'http://localhost:9200/test-elasticsearch-sink/_search?pretty'
```
[the remark this answers is not in the document]
Comment (joel-hamill) [reply docs/elasticsearch_connector.rst]: i will add the limitation to version 5.x.x

**Comment by rayokota** [comment]
This is fixed with #169

**Pull request merged**